### ISSUE TITLE
"Time played" for multiple games at the same time

### ISSUE THREAD

**Issue by MikkelHauge** [JosefNemec/Playnite#3408]
### Bug Description

basically i sometimes play two games at the same time.

and i noticed playnite stops marking the first game as "running" when i open the 2nd game.

i mean i guess for most of people its ok, but.. its a minor thing, clearly.

basically i had a game open (through an emulator) and then i opened hearthstone through battle.net and the "play" button for my emulated game turned from "running" to "play" even though the game was still running. 

there's been no crashing or other errors regarding this

thanks for an awesome piece of software. <3

### To Reproduce

basically i had a game open (through an emulator) and then i opened hearthstone through battle.net and the "play" button for my emulated game turned from "running" to "play" even though the game was still running.

### Diagnostics ID

5739b0e0-57f7-440b-a5e0-8529f7d53b5f

**Comment by MikkelHauge** [comment]
i should clarify - i opened hearthstone through playnite - which then open battle.net - which then opens hearthstone.

**Comment by JosefNemec** [comment]
Can't reproduce. Tracking of multiple games at the same time is already supported.

**Comment by JosefNemec** [comment]
If it specifically happens when battle.net game is started, then it's most likely some weird bug in battle.net plugin.
